string[] limiting_signal_monitor_names
int32[]  limiting_signal_monitor_actions
